static/src/js/components/CollectionAssociationForm/__tests__/CollectionAssociationForm.test.jsx

```diff
@@ -413,7 +412,7 @@ describe('CollectionAssociationForm', () => {
 
   describe('when updating a variable collection association', () => {
     describe('updating variable collection association results in a success', () => {
-      test('should update and navigate to collection-association', async () => {
+      test.skip('should update and navigate to collection-association', async () => {
```

Comment:
Why are we skipping?

Reply:
I should have deleted it, because we're not doing this using navigate anymore, its just a link